src/autoload.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
could be done by `psr/container` as well to get rid of `container-interop/container-interop` once and for all 🤷🏼‍♂️